test/test.js

```diff
@@ -797,4 +797,111 @@ describe('avltree-js tests', function() {
 			assert.deepStrictEqual(expectedOrder, actualOrder);
 		});
 	});
+	describe('copy', function () {
+		var tree;
+		beforeEach(function () {
+			tree = new AvlTree();
+		});
+		it('should do deep copy correctly', function () {
```

Comment:
Not sure I would call it a deep copy, to me it implies that the elements will be cloned as well.

Reply:
Thinking on it, you do have a point. For primitive objects, the definition might work, but yeah, for others, not so. Do you have a preference between cloning the actual object to align with how deep copy is defined or simply correcting the description of this test?